pkg/machine/applehv/machine.go

```diff
@@ -89,16 +88,9 @@ func (m *MacMachine) setGVProxyInfo(runtimeDir string) error {
 	if err != nil {
 		return err
 	}
-
-	gvProxySock, err := machine.NewMachineFile(filepath.Join(runtimeDir, "gvproxy.sock"), nil)
-	if err != nil {
-		return err
-	}
-
 	m.GvProxyPid = *gvProxyPid
-	m.GvProxySock = *gvProxySock
 
-	return nil
+	return machine.SetSocket(&m.GvProxySock, filepath.Join(runtimeDir, "gvproxy.sock"), nil)
```

Comment:
Old I know, and not necessary for this PR.  But it would be nice at some point to get a constant setup for this bare string.

Reply:
My thinking is that might work best after https://github.com/containers/podman/pull/20572 merges? It doesn't feel "right" to put it in `pkg/machine/config.go` (in my opinion)